packages/core/admin/admin/src/pages/MarketplacePage/index.js

```diff
@@ -106,60 +89,13 @@ const MarketPlacePage = () => {
             })}
           />
           <ContentLayout>
-            <StackCentered
-              size={0}
-              hasRadius
-              background="neutral0"
-              shadow="tableShadow"
-              paddingTop={10}
-              paddingBottom={10}
-            >
-              <Box paddingBottom={7}>
-                <Img
-                  alt={formatMessage({
-                    id: 'admin.pages.MarketPlacePage.illustration',
-                    defaultMessage: 'marketplace illustration',
-                  })}
-                  src={MarketplacePicture}
-                />
-              </Box>
-              <Typography variant="alpha">
-                {formatMessage({
-                  id: 'admin.pages.MarketPlacePage.coming-soon.1',
-                  defaultMessage: 'A new way to make Strapi awesome.',
-                })}
-              </Typography>
-              <Typography variant="alpha" textColor="primary700">
-                {formatMessage({
-                  id: 'admin.pages.MarketPlacePage.coming-soon.2',
-                  defaultMessage: 'A new way to make Strapi awesome.',
-                })}
-              </Typography>
-              <Flex maxWidth={pxToRem(580)} paddingTop={3}>
-                <CenterTypography variant="epsilon" textColor="neutral600">
-                  {formatMessage({
-                    id: 'admin.pages.MarketPlacePage.content.subtitle',
-                    defaultMessage:
-                      'The new marketplace will help you get more out of Strapi. We are working hard to offer the best experience to discover and install plugins.',
-                  })}
-                </CenterTypography>
-              </Flex>
-              <Stack paddingTop={6} horizontal size={2}>
-                {/* Temporarily hidden until we have the right URL for the link */}
-                {/* <LinkButton href="https://strapi.io/" size="L" variant="secondary">
-                  {formatMessage({
-                    id: 'admin.pages.MarketPlacePage.submit.plugin.link',
-                    defaultMessage: 'Submit your plugin',
-                  })}
-                </LinkButton> */}
-                <LinkButton href="https://strapi.io/blog/strapi-market-is-coming-soon" size="L">
-                  {formatMessage({
-                    id: 'admin.pages.MarketPlacePage.blog.link',
-                    defaultMessage: 'Read our blog post',
-                  })}
-                </LinkButton>
-              </Stack>
-            </StackCentered>
+            <Grid gap={4}>
+              {pluginsResponse.data.map(plugin => (
+                <GridItem col={4} s={6} xs={12} style={{ height: '100%' }} key={plugin.id}>
+                  <PluginCard plugin={plugin} />
+                </GridItem>
+              ))}
+            </Grid>
```

Comment:
Can we render this as a ordered list (`ol/li`)? Again for accessibility reasons ...

Reply:
Unfortunately when writing `<GridItem as="li">`, the "as" prop is only applied to [an inner child](https://github.com/strapi/design-system/blob/main/packages/strapi-design-system/src/Grid/GridItem.js#L25). So my DOM tree is

- ul (Grid with as="ul")
  - div (GridItem)
    - li (Box inside GridItem)

I'm not sure how I can fix that while still implementing the design system.